.github/issue_template/bug-report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: Report a bug in Skript
+
+---
+### Description
+<!--- What happened? --->
+
+### Steps to Reproduce
+<!--- What did you do to make it happen? Provide as accurate instructions as possible! --->
+
+### Expected Behavior
+<!--- What should have happened? --->
+
+### Errors / Screenshots
+<!--- Console errors and screenshots of visual bugs in game, if you have any --->
+
+<!---
+If you have console errors, copy them to a paste service which won't delete them.
+DON'T use Hastebin!
+--->
+
+<!--
+Screenshots of bugs visible in-game can also be attached.
+--->
+
+### Server Information
+* **Server version/platform:** <!-- /version -->
+* **Skript version:** <!-- /version Skript -->
```

Comment:
May want to include a note like: "'Latest' is not an appropriate version"

Reply:
Good point, we see this way too much.